spec/hyacinth/adapters/preservation_adapter/fedora3/system_spec.rb

```diff
@@ -7,16 +7,20 @@
   let(:location_uri) { "fedora3://" + object_pid }
   let(:pid_generator) { instance_double(PidGenerator) }
   let(:resource_dsid_overrides) { { 'master' => 'content', 'main' => 'content' } }
-  let(:rubydora_config) { Rails.application.config_for(:fedora) }
```

Comment:
`rubydora_config` was only used in the `let` statement below it, so that's why I removed this `let` statement.